src/entities/applet/ui/AppletCard.tsx

```diff
@@ -1,16 +1,23 @@
-import { Applet } from "../lib/applet.schema"
+import { useNavigate } from "react-router-dom"
 
+import { AppletDetailsDto } from "~/shared/api"
 import { CustomCard } from "~/shared/ui"
+import { ROUTES } from "~/shared/utils"
 
 interface AppletCardProps {
-  applet: Applet
+  applet: AppletDetailsDto
```

Comment:
The previous implementation was correct. We should not depend on API interfaces here, on the entity level

Reply:
Done